lib/models/liveValidationError.js

```diff
@@ -21,7 +21,7 @@
  */
 class LiveValidationError {
   constructor(code, message) {
-    this.code = code;
+    this.code = code.name;
```

Comment:
thanks Vishrut, makes sense, I went with the simplest approach for now and updated the callers to pass the name.